### ISSUE TITLE
Loading nexus files in Unity

### ISSUE THREAD

**Issue by TuomasK1989** [cnr-isti-vclab/nexus#117]
Hi, Would it be possible to render these Nexus files in Unity or is there any plans to develop this or is there any ready implementations?

**Comment by ponchio** [comment]
Not at the moment.
There is a plan to develop nexus for Unity: the Corto lib has been ported to unity, and hopefully in a few months, also  Nexus will be ported to Unity.

**Comment by TuomasK1989** [comment]
This sounds great. Couple months have passed and would like to inquire if there has been any progress on this or updated schedule? Will the code be available on this github page when ready? I would also be interested to do some testing and giving feedback for developer if it would help.

**Comment by ponchio** [comment]
Yep, I learned Unity, made some prelimiary tests (some performance issues to solve). Yes the code will be available on this page. 

Testing and feedback would be really appreciated!

**Comment by TuomasK1989** [comment]
Hi, Just wanted to know if there has been some progress porting nexus to unity?

**Comment by Mobeen-Invozone** [comment]
Hi, Any progress on this?
